lifeapp/src/components/Todo.js

```diff
@@ -0,0 +1,33 @@
+import React, { Component } from 'react';
+  import { ToastContainer, toast } from 'react-toastify';
+  import 'react-toastify/dist/ReactToastify.css';
+
+
+  const  TodoList = (props) =>{
+        
+    return props.name.map((todo,i) => {
+      // debugger
+      return (
+        <li value={todo} key={todo}  onClick={onRemoveItem}>
+        <ToastContainer />
+          {todo}
+        </li>
+      );
+    });
+    // onClick= {() => onRemoveItem(i)}
+  }
+  const onRemoveItem = (i) => {
+    // debugger
+    this.setState(state => {
```

Comment:
since this is a functional component it wouldnt be having the method `this.setState`